### PR TITLE
Eliminated 404s in V5 docs + nits

### PR DESCRIPTION
**Pull request by andreibratu**

### CODE REVIEW

File: fern/v5-pages/api-reference/errors.md [hunk 1]
```diff
@@ -14,17 +14,17 @@ Your request was improperly formatted or presented.
...
-<Accordion title="400 Bad request">
 Your API key is incorrect or missing, or your user does not have the rights to access the relevant resource.
 </Accordion>
 
 <Accordion title="404 Not found">
 The requested resource could not be located.
 </Accordion>
 
+<Accordion title="409 Conflict">
+Modifying the resource would leave it into an illegal state.
```
Comment (PeadarOhAodha) [inline fern/v5-pages/api-reference/errors.md:25]: typo

Comment (andreibratu) [reply fern/v5-pages/api-reference/errors.md]: ack fixed

File: fern/pages/guides/evaluate/evaluate-offline.mdx [hunk 2]
```diff
@@ -15,11 +15,7 @@ image: https://humanloop.com/assets/docs/social-image.png
 - Finally, you need at least a few Logs in your prompt. Use the **Editor** to generate some logs if you don't have any yet.
 
 <Info>
-  You need logs in your project because we will use these a source of test
-  datapoints for the dataset we will create. If you want to create arbitrary
-  test datapoints from scratch, see our guide to doing this from the API. We
-  will soon be updating the app to enable arbitrary test datapoint creation from
-  your browser.
+You need logs in your project because we will use these a source of test datapoints for the dataset we will create. If you want to create arbitrary test datapoints from scratch, see our guide to doing this from the API. We will soon be updating the app to enable arbitrary test datapoint creation from your browser.
```
Comment (PeadarOhAodha) [inline fern/pages/guides/evaluate/evaluate-offline.mdx:18]: typos

File: fern/pages/guides/experiments/run-experiment.mdx [hunk 3]
```diff
@@ -12,9 +12,7 @@ Experiments can be used to compare different prompt templates, different paramet
 - You have integrated `humanloop.complete_deployed()` or the `humanloop.chat_deployed()` endpoints, along with the `humanloop.feedback()` with the [API](https://www.postman.com/humanloop/workspace/humanloop) or [Python SDK](./generate-and-log-with-the-sdk).
 
 <Info>
-  This guide assumes you're are using an OpenAI model. If you want to use other
-  providers or your own model please also look at the [guide for running an
-  experiment with your own model provider](./use-your-own-model-provider).
+This guide assumes you're are using an OpenAI model. If you want to use other providers or your own model please also look at the [guide for running an experiment with your own model provider](./use-your-own-model-provider).
```
Comment (PeadarOhAodha) [inline fern/pages/guides/experiments/run-experiment.mdx:15]: typos

File: fern/v5-pages/concepts/flows.mdx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+subtitle: Flows centralise multiple Logs and allow you to evaluate them together.
```
Comment (PeadarOhAodha) [inline fern/v5-pages/concepts/flows.mdx:2]: 'centralise multiple logs' is not clear

File: fern/v5-pages/concepts/flows.mdx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+description: Discover how to instrument complex usecases such as RAG or language agents with Humanloop.
+---
+
+A Flow on Humanloop represents a virtual model of your AI app, collecting Logs created during an interaction session between the end user and your system. If your AI feature involves more than one File, Flows allow you to centralise their Logs in one UI view.
```
Comment (PeadarOhAodha) [inline fern/v5-pages/concepts/flows.mdx:6]: Think we need more clarity here, some of the english reads a little awkward 'interaction session between the end user and your system'

File: fern/v5-pages/concepts/flows.mdx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+## Using Flows
+
+A collection of Logs is called a Trace. The Logs of a Flow file represent the heads of these Traces to which Logs are appended. Furthermore, any Log inside the Trace can have other Logs under it.
```
Comment (PeadarOhAodha) [inline fern/v5-pages/concepts/flows.mdx:10]: Clarity here also needs to be improved.

It reads awkwardly

File: fern/v5-pages/concepts/flows.mdx [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+To use Flows, log a Trace under a Flow file and assign it an ID. If none is provided, Humanloop will assign one.
+
+<EndpointRequestSnippet endpoint="POST /flows/log" />
+
+When Logs are added to Humanloop, the Log can be coopted into a Trace by specifying the ID of the Trace in the request body. Furthermore, the new Log request can specify a trace parent, which nests it under another Log inside the Trace.
```
Comment (PeadarOhAodha) [inline fern/v5-pages/concepts/flows.mdx:16]: coopted is confusing. Keep the language simple. 

This section should also be improved in terms of clarity

File: fern/v5-pages/concepts/flows.mdx [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+Once all Logs have been added to the Trace, the Trace must be marked as complete. This allows Monitoring Evaluators to pick up the Trace and evaluate it.
+
+## Versioning
+
+Versioning a Flow is done by modifying the attributes object of the Kernel. The attributes object supports arbitrary metadata inside it, and every change creates a new Version of the Flow. This flexibility allows you to instrument complex usecases.
```
Comment (PeadarOhAodha) [inline fern/v5-pages/concepts/flows.mdx:22]: Kernel is not a user facing concept. Remove it

Comment (PeadarOhAodha) [reply fern/v5-pages/concepts/flows.mdx:22]: Also talking about metadata and attributes in the same paragraph feels wrong

File: fern/v5-pages/concepts/flows.mdx [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+For example, a chatbot can be represented as a state machine with multiple paths corresponding to the chatbot's capabilities. A user might only interact with some paths in the graph, resulting in heterogeneous Traces for the same Flow Version.
```
Comment (PeadarOhAodha) [inline fern/v5-pages/concepts/flows.mdx:24]: state machine
heterogeneous 

Too much jargon, can be made way way simpler

File: fern/v5-pages/concepts/flows.mdx (hunk 8 above)
Comment (andreibratu) [reply fern/v5-pages/concepts/flows.mdx:22]: ack fixed

File: fern/v5-pages/concepts/flows.mdx (hunk 9 above)
Comment (andreibratu) [reply fern/v5-pages/concepts/flows.mdx:24]: ack fixed

File: fern/v5-pages/concepts/flows.mdx (hunk 6 above)
Comment (andreibratu) [reply fern/v5-pages/concepts/flows.mdx:10]: ack changed

File: fern/v5-pages/concepts/flows.mdx (hunk 5 above)
Comment (andreibratu) [reply fern/v5-pages/concepts/flows.mdx:6]: ack changed

File: fern/v5-pages/concepts/flows.mdx (hunk 4 above)
Comment (andreibratu) [reply fern/v5-pages/concepts/flows.mdx:2]: 'aggregate' perhaps?

File: fern/pages/guides/experiments/run-experiment.mdx (hunk 3 above)
Comment (andreibratu) [reply fern/pages/guides/experiments/run-experiment.mdx:15]: fixed

File: fern/pages/guides/evaluate/evaluate-offline.mdx (hunk 2 above)
Comment (andreibratu) [reply fern/pages/guides/evaluate/evaluate-offline.mdx:18]: ack changed

File: fern/v5-pages/concepts/flows.mdx [hunk 10]
```diff
@@ -0,0 +1,24 @@
+---
+subtitle: Flows combine multiple Logs to enable joint evaluation.
+description: Learn how to use Humanloop to manage and evaluate complex AI use cases such as Retrieval-Augmented Generation (RAG) or language agents.
+---
+
+A Flow in Humanloop helps you capture and manage Logs representing individual user interactions with your AI system. If your AI application involves multiple steps or components, Flows group these Logs together, providing a complete view of each interaction for easier analysis.
```
Comment (PeadarOhAodha) [inline fern/v5-pages/concepts/flows.mdx:6]: this is still weird.

Why are we relating Flows with individual user interactions

File: fern/v5-pages/concepts/flows.mdx [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+## How to Use Flows
+
+Flows allow you to organize multiple Logs into a Trace. Each Log within a Flow file serves as the starting point of a Trace.
+
+To use Flows, add a Log under a Flow file and assign it a `trace_id`. If you don't provide an ID, Humanloop will generate one for you.
```
Comment (PeadarOhAodha) [inline fern/v5-pages/concepts/flows.mdx:12]: This is wrong its no longer trace_id

**Review by PeadarOhAodha — APPROVED** [review]
@andreibratu ive removed Flows concepts as it still need a lot of work to get it to the clarity we need

**Pull request merged**